src/openapi/streaming/connection/transport/signalr-core-transport.ts

```diff
@@ -0,0 +1,623 @@
+import log from '../../../../log';
+import * as transportTypes from '../transportTypes';
+import * as constants from '../constants';
+import type {
+    ConnectionState,
+    TransportTypes,
+    StreamingMessage,
+} from '../../types';
+import type {
+    StreamingTransportOptions,
+    StreamingTransportInterface,
+    StreamingData,
+} from '../types';
+import type SignalR from '@microsoft/signalr';
+
+declare global {
+    interface Window {
+        signalrCore: typeof SignalR;
+    }
+}
+
+type Callback = (...args: any[]) => any;

```

Comment:
return any or void?
The use of generic callbacks - was that a time issue, rather than typing each callback?

Reply:
I cleaned up the callback mess. Thanks for pointing this out!